### PR TITLE
Added messages notifications.

### PR DESCRIPTION
**Pull request by andresg4**
#### Description:

* Add notifications when messages are sent.
---
#### Trello cards addressed:

* https://trello.com/c/zvV99f2P
---
#### Tasks Done

- [X] Added jobs to perform new messages and match targets notifications.
- [X] Added send notification after message is created.

### CODE REVIEW

File: app/jobs/notify_new_message_job.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+class NotifyNewMessageJob < ApplicationJob
+  queue_as :default
+  retry_on RuntimeError
+
+  def perform(current_user, devices_id, message)
+    NotificationService.new(current_user).notify_user(
+      devices_id,
+      'You have a new message',
```
Comment (Alxzu) [inline app/jobs/notify_new_message_job.rb]: Make this I18n

File: app/jobs/notify_match_target_job.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
+class NotifyMatchTargetJob < ApplicationJob
+  queue_as :default
+  retry_on RuntimeError
+
+  def perform(current_user, devices_id)
+    NotificationService.new(current_user).notify_user(
+      devices_id,
+      'You have a new match',
```
Comment (Alxzu) [inline app/jobs/notify_match_target_job.rb]: Make this I18n

**Review by Alxzu — APPROVED** [review]
Just a small comment, seems good to go. Nice work changing the service into a job! 💪

**Review by jdrosales17 — APPROVED** [review]
Nice!!

**Pull request merged**